code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -30,51 +30,39 @@
 		if ("small")
 			new /obj/item/weapon/tank/emergency/oxygen(src)
 			new /obj/item/weapon/tank/emergency/oxygen(src)
+			new /obj/item/weapon/rig/emergency(src)
 			new /obj/item/clothing/mask/breath(src)
 			new /obj/item/clothing/mask/breath(src)
-			new /obj/item/clothing/suit/space/void(src)
-			new /obj/item/clothing/head/helmet/space/void (src)
-			if (prob(70))
-				new /obj/item/weapon/tool/tape_roll(src)
```

Comment:
tape rolls being present in emergency lockers is important, as they can be used for breach patching

Reply:
Apologies on this one, thought it was the department "Do not cross" tape. I'll put this back in.